src/lib/nftClaimer/utils.ts

```diff
@@ -0,0 +1,131 @@
+import { gql, ApolloClient, InMemoryCache, HttpLink } from '@apollo/client/core';
+import fetch from 'cross-fetch';
+import snapshot from '@snapshot-labs/snapshot.js';
+import { CID } from 'multiformats/cid';
+import { Wallet } from '@ethersproject/wallet';
+import { getAddress, isAddress } from '@ethersproject/address';
+import { BigNumber } from '@ethersproject/bignumber';
+import type { Proposal, Space } from '../../helpers/snapshot';
+
+const requiredEnvKeys = [
+  'NFT_CLAIMER_PRIVATE_KEY',
+  'NFT_CLAIMER_NETWORK',
+  'NFT_CLAIMER_DEPLOY_VERIFYING_CONTRACT',
+  'NFT_CLAIMER_DEPLOY_IMPLEMENTATION_ADDRESS',
+  'NFT_CLAIMER_DEPLOY_INITIALIZE_SELECTOR',
+  'NFT_CLAIMER_SUBGRAPH_URL'
+];
+
+const HUB_NETWORK = process.env.NETWORK || '1';
+
+const missingEnvKeys: string[] = [];
+requiredEnvKeys.forEach(key => {
+  if (!process.env[key]) {
+    missingEnvKeys.push(key);
+  }
+});
+
+if (missingEnvKeys.length > 0) {
+  throw new Error(
+    `NFT Claimer not configured properly, missing env keys: ${missingEnvKeys.join(', ')}`
+  );
+}
+
+export const signer = new Wallet(process.env.NFT_CLAIMER_PRIVATE_KEY as string);
+
+async function mintingAllowed(space: Space) {
+  return (await getSpaceCollection(space.id)).enabled;
+}
+
+export async function validateSpace(address: string, space: Space | null) {
+  if (!space) {
+    throw new Error('RECORD_NOT_FOUND');
+  }
+
+  if (
+    (process.env.NFT_CLAIMER_NETWORK !== '5' || process.env.NODE_ENV === 'test') &&
```

Comment:
we need to check `process.env.NODE_ENV` here for sure, we can instead pass different values in the tests, right?

Anyway, In `deploy.test.ts` we already mock this function?
https://github.com/snapshot-labs/snapshot-sidekick/blob/fix-nft-claimer-deploy-signing/test/unit/lib/nftClaimer/deploy.test.ts#L24
so maybe we don't need this check?

Reply:
This check is not to accomodate test, but the live version.

We do want this validation to be checked on test, but not on live (and not the opposite), to be able to deploy contract without being the space owner.